docs/homebridge.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+**Installation of this script has been deprecated in favor of the
+[`homekit` component](homekit_component)**
+
+## Remove
```

Comment:
Maybe add something like.
"This script only removes a previously installed installation of homebridge" or similar to the Description.

Reply:
I agree with this :)